### PR TITLE
[Ready for review] Upload notes from Sprint 2 backlog refinement meeting 31-03-2020

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
New markdown document for meeting notes for backlog refinement meeting on the 31st March.

### CODE REVIEW

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* Georgia Aitkenhead/GeorgiaHCA
+* Ellen Devereux/EllenDevereux
+* Joe Collins/JoeC7756
+* Conor Henderson
+* Harrison Cannon
+* Jonathan Kerfoot
+* Ryan Galley
+* Jake Wych
+* Emilia Driscoll 
+* Christopher Legg
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:14]: Is everyone comfortable having their full names in the GitHub repository? Could replace with initials and/or GitHub usernames? @fjThomasStanley

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Harrison Cannon
+* Jonathan Kerfoot
+* Ryan Galley
+* Jake Wych
+* Emilia Driscoll 
+* Christopher Legg 
+
+### Apologies: 
+
+* Kirstie Whitaker/KirstieJane - urgent coronavirus work commitments
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:18]:
```suggestion
* Kirstie Whitaker / @KirstieJane - work commitments
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Saima Butt
+
+### Useful Links:
+
+* Business Requirements: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/94
+* Agile and Open Source Working: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/97
+* Create Demo Video: https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/116
+* Sprint Meetings Documentation: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/123
+* Wireframes: https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/103
+* Documentation on Sprint Progress:  https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/115
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:28]:
```suggestion
* Documentation on Sprint Progress: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/115
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* Securely transfers data from users categorised as usable for research and/or usable for public view, or neither, to the Open Humans backend file management system.
+* Allows moderation of input before it is made public.
+* Presents users with their own data and consent decisions.
+* Displays experiences that have been consented for public view to any visitor of the website.
+
+### Backlog Prioritisation: 
+
+* Fujitsu team to review and agree business requirements: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/94
+* Review and agree agile open source working process: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/97
+* Create sprint report and demo Video https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/116
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:45]:
```suggestion
* Create sprint report and demo video https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/116
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+* Clearly and accessibly informs users about the project, the platform, what it is for and how to use it.
+* Registers users and collects necessary demographic information, such as connection to autism, and is compliant with the UK Data Protection Act.
+* Takes input from users reporting their experiences navigating different sensory environments.
+* Securely transfers data from users categorised as usable for research and/or usable for public view, or neither, to the Open Humans backend file management system.
+* Allows moderation of input before it is made public.
+* Presents users with their own data and consent decisions.
+* Displays experiences that have been consented for public view to any visitor of the website.
+
+### Backlog Prioritisation:
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:41]: Can we add some dates for sprint 2 here?

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+* Presents users with their own data and consent decisions.
+* Displays experiences that have been consented for public view to any visitor of the website.
+
+### Backlog Prioritisation: 
+
+* Fujitsu team to review and agree business requirements: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/94
+* Review and agree agile open source working process: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/97
+* Create sprint report and demo Video https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/116
+    * Create documentation on wireframes, sprint report (preparatory work) provide links on GitHub to where hosted - record of decisions made and processes done - co-working session Georgia and Chris to put everything on GitHub
+* Complete Sprint Meetings Documentation: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/123
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:47]:
```suggestion
* Complete sprint meetings documentation: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/123
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+* Document backlog definitions (user stories and goals): https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/103
+* Link 1:1 user testing documentation to ways of working: https://github.com/alan-turing-institute/AutisticaCitizenScience/pull/115
+    Speak to community members to clarify structure they would like to be followed - document conversations - make this part of the     review of the workflow documentation pull requests (linked above)
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:50]:
```suggestion
  * Speak to community members to clarify structure they would like to be followed - document conversations - make this part of the review of the workflow documentation pull requests (linked above)
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+* Drill down into business requirements to clarify acceptance criteria: speak to the community
+* (How): Define strategy for how will go about talking to community to gather info for acceptance criteria - process of speaking to community can happen at the same time - create in conjunction with the community so community driven 
+
+### GA Actions: 
+
+* Publish summaries on GitHub (put into markdown and incorporate SI's suggestions)
+* Publish notes from meetings with TA, JS, and EH on GitHub
+* Go through Repo and organise - pull requests, labels, issues, milestones
+* Write up meeting on 27th put into markdown 
+* Put Fujitsu team in touch with community members -
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:61]:
```suggestion
* Put Fujitsu team in touch with community members
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:84]: Suggest delete these lines.

```suggestion
```

File: project-management/backlog-refinement-meetings/03-31-2020.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+* Publish notes from meetings with TA, JS, and EH on GitHub
+* Go through Repo and organise - pull requests, labels, issues, milestones
+* Write up meeting on 27th put into markdown 
+* Put Fujitsu team in touch with community members - 
+* Set up co-working session with CL 
+
+### Meeting Minutes
+
+* Focus on documentation, process, and defining acceptance criteria before development work begins
+* Full sprint not yet in progress - preparatory work has occured, but interative design not yet in place - but a full sprint report will be published
```
Comment (KirstieJane) [inline project-management/backlog-refinement-meetings/03-31-2020.md:67]: Can we add a date to this? Published at the end of sprint 2 or retrospectively for sprint 1? What about the video?

**Comment by GeorgiaHCA** [comment]
Thanks for the editing @KirstieJane ! :-D - I've included your suggestions. @SaimaButt - it would be amazing if your team could review the minutes to make sure they reflect the meeting/your expeectations, and respond to Kirstie's unresolved comment :-) Thanks so much!!

**Comment by KirstieJane** [comment]
This work has been addressed in #156. Once that is ready for review (when it doesn't say [WIP] in the title) please check that PR. Once it is merged this PR can close.

**Comment by EllenDevereux** [comment]
#156 has been merged.

**Comment by GeorgiaHCA** [comment]
Closing this as #156 has been merged.

**Pull request closed without merging**